app/models/user.rb

```diff
@@ -92,7 +92,6 @@ def create_confirmation_token
 
   def email_activate
     self.email_confirmed = true
-    self.confirm_token = nil
     save!(validate: false)
```

Comment:
Tricky, I had missed that you removed this.  When rolling out this code, it's important to put back a `confirm_token` for existing users.  That was the root cause for this error: https://github.com/DevProgress/HillaryBNB/issues/88